app/serializers/basic_category_serializer.rb

```diff
@@ -38,4 +38,7 @@ def include_can_edit?
     scope && scope.can_edit?(object)
   end
 
+  def notification_level
+    object.notification_level ? object.notification_level : CategoryUser.where(user: object.user, category: object).first.try(:notification_level)
```

Comment:
I think this can be simplified to `object.notification_level || CategoryUser.where....` :smile: